plugin/storage/integration/fixtures/traces/multispottag_opname_trace.json

```diff
@@ -4,7 +4,6 @@
       "traceId": "AAAAAAAAAAAAAAAAAAAAFw==",
       "spanId": "AAAAAAAAAAQ=",
       "operationName": "query17-operation",
-      "references": [],
```

Comment:
is there a dependency in this PR on these changes?  If so, could they be made separately in a different PR?